tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -549,3 +549,30 @@ def test_invalid_attr_not_allowed(self, test_client, session):
         response = test_client.post("/public/assets/events", json=event_invalid_payload)
 
         assert response.status_code == 422
+
+
+class TestDeleteAssetQueuedEvents(TestQueuedEventEndpoint):
+    @pytest.mark.usefixtures("time_freezer")
+    def test_should_respond_204(self, test_client, session, create_dummy_dag):
+        dag, _ = create_dummy_dag()
+        dag_id = dag.dag_id
+        uri = "s3://bucket/key/1"
+        self.create_assets(session=session, num=1)
+        asset_id = 1
+        self._create_asset_dag_run_queues(dag_id, asset_id, session)
+
+        response = test_client.delete(
+            f"/public/assets/queuedEvent/{uri}",
```

Comment:
Maybe we should take the opportunity to `assets/queudEvents` to be consistent with other routes, especially `assets/events`

Reply:
Follow up PR, resolving